radio/src/pulses/pxx_arm.cpp

```diff
@@ -378,18 +378,19 @@ void setupPulsesPXX(uint8_t port)
 /* Ext. flag (holds antenna selection on Horus internal module, 0x00 otherwise) */
 #if defined(PCBHORUS)
   if (port == INTERNAL_MODULE) {
-    extra_flags |= g_model.moduleData[port].pxx.external_antenna;
+    extra_flags |= (g_model.moduleData[port].pxx.external_antenna << 0);
   }
 #endif
 #if defined(BINDING_OPTIONS)
-  extra_flags |= g_model.moduleData[port].pxx.receiver_telem_off << 1;
-  extra_flags |= g_model.moduleData[port].pxx.receiver_channel_9_16 << 2;
+  extra_flags |= (g_model.moduleData[port].pxx.receiver_telem_off << 1);
+  extra_flags |= (g_model.moduleData[port].pxx.receiver_channel_9_16 << 2);
 #endif
   if (IS_MODULE_R9M(port)) {
-    extra_flags |= g_model.moduleData[port].pxx.power << 3;
-    // Disable s.port if internal module is active
-    if (IS_TELEMETRY_INTERNAL_MODULE || !g_model.moduleData[port].pxx.sport_out)
-      extra_flags |=  (1<< 5);
+    extra_flags |= (max(g_model.moduleData[port].pxx.power, IS_MODULE_R9M_FCC(port) ? (uint8_t)R9M_FCC_POWER_MAX : (uint8_t)R9M_LBT_POWER_MAX) << 3);
```

Comment:
Actually... where is the proper power set for EU module?  Or are those bits ignored anyway since it's based on telem on/off?

Reply:
yes they are ignored

Reply:
that also removes the sport out for FCC module, does it no longer respect that bit?

Reply:
sorry it is only moved, ignore my comment.